src/components/landing/CommunityListingHeader.js

```diff
@@ -0,0 +1,98 @@
+import CustomizableAvatar from "../common/customavatar/CustomizableAvatar";
+import Link from "next/link";
+import Image from "next/image";
+
+const CommunityHeader = ({ listing }) => {
+  const photo = "/community dashoard  img 1.jpg";
+  const Manager_Avatar = "/avarar 1.png";
+  const AVATAR_WIDTH = 50;
+  const AVATAR_HEIGHT = 50;
+  const SmallAvatar_WIDTH = 20;
+  const SmallAvatar_HEIGHT = 20;
+  const LOGO_WIDTH = 20;
+  const LOGO_HEIGHT = 20;
+  return (
+    <header className="">
+      <div className=" bg-primary-lite p-5 md:flex lg:flex items-center justify-between px-10">
+        <article className="flex items-center pb-4">
+          <CustomizableAvatar
```

Comment:
For this MVP, let's get rid of this status image that appears before the Community Name. The app MVP does not receive images (like logos) from managers when they create their listing, and I do not see the point of having a static image that is the same for all communities.